### PR TITLE
Add UBI Docker images

### PR DESCRIPTION
**Pull request by jasonodonnell**
This adds support for UBI flavored container images. All UBI related files are in a new directory `ubi/` in the root directory of this project.

UBI-minimal is being used as the base image to shave a little bit off the container image size.

### CODE REVIEW

File: ubi/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+# Expose the file directory as a volume since there's potentially long-running
+# state in there
+VOLUME /vault/file
+
+# 8200/tcp is the primary interface that applications use to interact with
+# Vault.
+EXPOSE 8200
+
+# The entry point script uses dumb-init as the top-level process to reap any
```
Comment (tomhjp) [inline ubi/Dockerfile:64]: This isn't true for UBI is it?

**Review by tomhjp — APPROVED** [review]
LGTM - just one comment that looks outdated to me

File: ubi/Dockerfile (hunk 1 above)
Comment (jasonodonnell) [reply ubi/Dockerfile]: Removed, thanks!

**Review by Valarissa — APPROVED** [review]
LGTM as well~!

**Review by sarahethompson — APPROVED** [review]

**Pull request merged**